packages/react-components/react-alert/README.md

```diff
@@ -1,6 +1,6 @@
 # @fluentui/react-alert
 
-**React Alert component for [Fluent UI React](https://developer.microsoft.com/en-us/fluentui)**
+**React Alert component for [Fluent UI React](https://react.fluentui.dev)**
 
 > WIP 🚧 - These are not production-ready components as we are still in a Beta phase.
```

Comment:
should this be removed as well ?

Reply:
No. I removed the message only from the components which are exported as stable in react-components. `react-alert` is still unstable.